toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -449,6 +449,18 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 		}
 	}
 
+	// Add groups
```

Comment:
I think we will still need to add an install of shadow-utils early here, maybe include it with filesystem on line 438.
Something like
```go
if len(config.Users) > 0 || len(config.Groups) > 0 {
    	shadowInstalled, err := TdnfInstallWithProgress(filesystemPkg, installRoot, packagesInstalled, totalPackages, true)
	if err != nil {
		return
	}
        packagesInstalled += shadowInstalled
}